sql/rounds.js

```diff
@@ -19,7 +19,7 @@ var RoundsSql = {
 
   updateBlockId: 'UPDATE mem_accounts SET "blockId" = ${newId} WHERE "blockId" = ${oldId};',
 
-  summedRound: 'WITH round_blocks as (SELECT ((${round}-1)*${activeDelegates})+1 as min, ${round}*${activeDelegates} as max) SELECT SUM(b."totalFee")::bigint AS "fees", ARRAY_AGG(b."reward") AS "rewards", ARRAY_AGG(ENCODE(b."generatorPublicKey", \'hex\')) AS "delegates" FROM blocks b WHERE b.height BETWEEN (SELECT min FROM round_blocks) AND (SELECT max FROM round_blocks)'
+  summedRound: 'WITH round_blocks as (SELECT ((${round} - 1) * ${activeDelegates}) + 1 AS min, ${round} * ${activeDelegates} AS max) SELECT SUM(b."totalFee")::bigint AS "fees", ARRAY_AGG(b."reward") AS "rewards", ARRAY_AGG(ENCODE(b."generatorPublicKey", \'hex\')) AS "delegates" FROM blocks b WHERE b.height BETWEEN (SELECT min FROM round_blocks) AND (SELECT max FROM round_blocks)'
```

Comment:
I not sure if we need to sort results here, is order important in aggregate?

We can also simplify that query to:
```
SELECT SUM(b."totalFee")::bigint AS "fees", ARRAY_AGG(b."reward") AS "rewards", ARRAY_AGG(ENCODE(b."generatorPublicKey", 'hex')) AS "delegates" FROM blocks b WHERE CEIL(b.height / ${activeDelegates}::float)::int = ${round};
```
Because we already have index on rounds - performance is the same, but new one is more clear.